WcaOnRails/app/models/user.rb

```diff
@@ -293,6 +285,44 @@ def after_confirmation
     end
   end
 
+  def software_team?
+    team_member?('software') != false
+  end
+
+  def results_team?
+    team_member?('results') != false
+  end
+
+  def wrc_team?
+    team_member?('wrc') != false
+  end
+
+  def wdc_team?
+    team_member?('wdc') != false
+  end
+
+  def team_member?(team)
+    member = self.team_members.find_by_team_id( self.teams.find_by_friendly_id(team) )
+    if member && (member.end_date == nil || member.end_date >= Date.today)
```

Comment:
This looks like another place where having a method like `current_member?` on team_members would be nice.

Can you explain how this works with people who were on and off teams? It looks like you're only looking at one of their team_members, rather than all of them.